browser/css/toolbar.css

```diff
@@ -1589,8 +1589,8 @@ menu-entry-with-icon.padding-left + menu-entry-icon.width */
 
 #userListHeader .avatar-img,
 #userlist-dropdown .avatar-img {
-	width: var(--btn-size);
-	height: var(--btn-size);
+	width: var(--btn-img-size-s);
+	height: var(--btn-img-size-s);
```

Comment:
why don't we use the --btn-img-size at least for the `#userListHeader .avatar-img` I still see missing the box-sizing